examples/12_use_myarm.md

```diff
@@ -0,0 +1,90 @@
+This tutorial explains how to use [Moss v1](https://github.com/jess-moss/moss-robot-arms) with LeRobot.
```

Comment:
TODO: Re-follow tutorial after all changes are done and validate it all works as described.

Reply:
Tested- everything is working!